Neos.Media.Browser/Configuration/Policy.yaml

```diff
@@ -16,6 +16,7 @@ privilegeTargets:
       matcher: 'method(Neos\Media\Browser\Controller\AssetProxyController->(index|import)Action())'
 
     'Neos.Media.Browser:ManageAssetCollections':
+      label: Manage Asset Collections
```

Comment:
Is it possible to use `i18n` labels here?

Reply:
Not yet